notebooks/Notebook1.md

```diff
@@ -2,33 +2,71 @@
 
 ## Last week's critique
 
-**TODO:** Fill in this part with a summary and reflection on the critique you received for
-last week's work. Answer questions such as:  How, specifically, did the feedback help
-improve the project? Did the feedback point out or offer something you hadn't considered?
-Did it help you make a design decision? Was it helpful in addressing the most pressing
-issues in your project? How will you incorporate the feedback into your work? Will you
-change something about the design, implementation, or evaluation as a result?
+The main way I worked with the feedback was by taking the suggestions of similar examples and doing more research into those.
+Since one comment was about it seeming like Prezi, I explored various example Prezi presentations and, although I do like that they are more fluid with everything being spatially connected, their structure is still relatively restrictive in my opinion with their fixed linear flow and hierarchical branching.
+So I may borrow some of the Prezi features, but hope to offer more beyond their structure.
+I looked at interactive videos as well and their capabilities are more in line with what I want to create, but I don't want my presentations to need a video base and want them to be more aimed toward an output that someone could use to present rather than something someone would explore on their own.
```

Comment:
If the user doesn't assign transitions to the elements, is there going to be a default transition style. I think the user should still be able to make a presentation without having to add transitions.